modules/highgui/src/window_cocoa.mm

```diff
@@ -876,12 +792,10 @@ void setWindowTitle_COCOA(const cv::String& winname, const cv::String& title)
     if (window == NULL)
         CV_Error(cv::Error::StsNullPtr, "NULL window");
 
-    NSAutoreleasePool* localpool = [[NSAutoreleasePool alloc] init];
-
-    NSString *windowTitle = [NSString stringWithFormat:@"%s", title.c_str()];
-    [window setTitle:windowTitle];
-
-    [localpool drain];
+    @autoreleasepool{
```

Comment:
I think `@autoreleasepool` block should include the whole function body, not only `NSString` creation